screens/FoodScreen.js

```diff
@@ -1,82 +1,113 @@
 import { Ionicons } from "@expo/vector-icons";
 import * as WebBrowser from "expo-web-browser";
 import React from "react";
-import { StyleSheet, Text, View } from "react-native";
-import { RectButton, ScrollView } from "react-native-gesture-handler";
-
+import Colors from "../constants/Colors";
+import { LinearGradient } from "expo-linear-gradient";
+import { Button, StyleSheet, Text, View } from "react-native";
+import { ScrollView } from "react-native-gesture-handler";
+import FoodMenu from "../components/FoodMenu";
+import MealPlan from "../components/MealPlan";
 export default function FoodScreen() {
-  return (
-    <ScrollView
-      style={styles.container}
-      contentContainerStyle={styles.contentContainer}
-    >
-      <OptionButton
-        icon="md-school"
-        label="Read the Expo documentation"
-        onPress={() => WebBrowser.openBrowserAsync("https://docs.expo.io")}
-      />
-
-      <OptionButton
-        icon="md-compass"
-        label="Read the React Navigation documentation"
-        onPress={() =>
-          WebBrowser.openBrowserAsync("https://reactnavigation.org")
-        }
-      />
 
-      <OptionButton
-        icon="ios-chatboxes"
-        label="Ask a question on the forums"
-        onPress={() => WebBrowser.openBrowserAsync("https://forums.expo.io")}
-        isLastOption
-      />
-    </ScrollView>
-  );
-}
 
-function OptionButton({ icon, label, onPress, isLastOption }) {
   return (
-    <RectButton
-      style={[styles.option, isLastOption && styles.lastOption]}
-      onPress={onPress}
-    >
-      <View style={{ flexDirection: "row" }}>
-        <View style={styles.optionIconContainer}>
-          <Ionicons name={icon} size={22} color="rgba(0,0,0,0.35)" />
-        </View>
-        <View style={styles.optionTextContainer}>
-          <Text style={styles.optionText}>{label}</Text>
-        </View>
-      </View>
-    </RectButton>
+    <View style={styles.container}>
+      <ScrollView contentContainerStyle={{ flex: 1 }}>

```

Comment:
This shouldn't be a ScrollView. Our design does not require a scrollview on this page.